op-node/p2p/sync.go

```diff
@@ -313,17 +316,23 @@ func (s *SyncClient) Close() error {
 	return nil
 }
 
-func (s *SyncClient) RequestL2Range(ctx context.Context, start, end eth.L2BlockRef) error {
+func (s *SyncClient) RequestL2Range(ctx context.Context, start, end eth.L2BlockRef) (uint64, error) {
```

Comment:
Added the return type for the `rangeReqId`, which makes this code much easier to test.